sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,588 @@
+# Two-Clients Approach
+
+## APIs:
+class SeviceBusSenderClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        entity_name : str,
+        credential : TokenCredential,
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = None,
+        retry_total : int = 3,
+    ) -> None:
+    @classmethod
+    def from_queue(
+        cls,
+        fully_qualified_namepsace : str,
+        queue_name : str,
+        credential : TokenCredential,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+    @classmethod
+    def from_topic(
+        cls,
+        fully_qualified_namespace : str,
+        topic_name : str,
+        credential : TokenCredential,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        entity_name : str = None,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, Any]:
+
+    def list_session_ids(self) -> List[str]:
```

Comment:
Would there be any benefit to having this as `list_sessions() -> List[Session]` instead?

Reply:
I guess that would only make sense if that session could then be passed into a Receiver.... so probably no benefit....

Reply:
I think there's one question needed to be answered first -- Is it a bit weird that a 
 `SessionReceiver` (which connect to a specific session) has the api to list all session ids?

Reply:
as per discussion, remove this method from the client